lighthouse-core/driver/gatherers/offline.js

```diff
@@ -38,23 +38,25 @@ const requestPage = function() {
 
 class Offline extends Gather {
 
-  static goOffline(driver) {
-    return driver.sendCommand('Network.emulateNetworkConditions', {
-      offline: true,
+  static config(opts) {
```

Comment:
sure but we're not using destructuring as we need to support node v4 w/ --harmony